crypto/doc/CHANGELOG_oberon.rst

```diff
@@ -9,6 +9,96 @@ Changelog - nrf_oberon
 
 All notable changes to this project are documented in this file.
 
+nrf_oberon - 3.0.15
+*******************
+
+New version of the nrf_oberon library with the following changes.
+
+Added
+=====
+
+* Added missing ocrypto functions for SECP160 curves which broke some configurations.
```

Comment:
The grammatical fix here depends on the intended meaning.

There should always be a comma after the word before `which`, which in this case would read, "SECP160  curves, which broke some configurations." 

However, I read this construction as meaning that adding the missing functions broke some configurations.

If this is not the intended meaning, use:

"SECP160 curves that broke..."

I read this as: missing functions broke some configurations, and adding them fixes it.

Reply:
I was intended to write that the fact that we had the missing functions broke some configurations. Now that I added them back they should be fixed. So I updated the text based on your second suggestion.